reducers/index.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
+import apiReducer from './apiReducer';
+
+export default combineReducers({
+  apiReducer,
```

Comment:
Which apiReducer is this? Please follow proper naming convention. If reducer is storing sports list data, then name it like sportsListReducer.